io_scene_nif/modules/object/object_export.py

```diff
@@ -312,7 +312,7 @@ def export_node(self, b_obj, n_parent):
         # export object animation
         if has_anim:
             self.nif_export.animationhelper.export_keyframes(node, b_obj)
-            self.nif_export.animationhelper.object_animation.export_object_vis_controller(node, b_obj)
+            self.nif_export.animationhelper.obj_anim.export_object_vis_controller(node, b_obj)
```

Comment:
Note to self, to rename this back to object_

Reply:
Ah yeah I was wondering if you had a new convention.
Actually I would suggest the following:
 self.nif_export.animationhelper.object.export_visibility_controller()...
So we cut down the reduncancy a bit

Reply:
Yep, sounds good, though eventually the aim is for this class to create the instance of animationhelper or even the specific object animation reference directly.

Reply:
So object_export would create its ObjectAnimation instance as needed? Yeah that seems good

Reply:
Yeah, we should get to the situation where these things are isolated.